package.json

```diff
@@ -1,86 +1,4 @@
 {
-  "name": "@google/dscc-gen",
-  "version": "2.0.19",
-  "repository": "googledatastudio/dscc-gen",
-  "description": "Create component & connector projects with sane defaults.",
-  "main": "build/index.js",
-  "bin": "build/index.js",
-  "engines": {
-    "node": ">=7.10.0",
-    "npm": ">=5.2.0"
-  },
-  "author": "Matt Hamrick <matthewjhamrick@gmail.com>",
-  "license": "Apache-2.0",
-  "jest": {
-    "transform": {
-      "^.+\\.tsx?$": "ts-jest"
-    },
-    "testRegex": "(/__tests__/.*|(\\.|/)(test|spec))\\.tsx?$",
-    "moduleFileExtensions": [
-      "ts",
-      "tsx",
-      "js",
-      "jsx",
-      "json",
-      "node"
-    ]
-  },
-  "prettier": {
-    "singleQuote": true,
-    "trailingComma": "es5",
-    "bracketSpacing": false,
-    "arrowParens": "always"
-  },
-  "scripts": {
-    "build": "tsc",
-    "build:check": "tsc --noEmit",
-    "build:watch": "tsc --watch",
-    "test": "jest",
-    "test:watch": "jest --watchAll --coverage",
-    "lint": "tslint -c tslint.json -p tsconfig.json \"src/**/*.ts\" \"test/**/*.ts\"",
-    "prettier:base": "prettier \"src/**/*.ts\" \"templates/**/*.js\" \"templates/**/*.ts\" \"test/**/*.ts\"",
-    "prettier:check": "yarn run prettier:base --check",
-    "prettier": "yarn run prettier:base --write",
-    "travis": "yarn prettier:check && yarn lint && yarn build:check && yarn test",
-    "fix": "yarn prettier && yarn lint --fix"
-  },
-  "dependencies": {
-    "@google/clasp": "^2.2.1",
-    "@types/mz": "^0.0.32",
-    "@types/shelljs": "^0.8.5",
-    "argparse": "^1.0.10",
-    "bluebird": "^3.5.5",
-    "chalk": "^2.4.2",
-    "clear": "^0.1.0",
-    "cli-spinner": "^0.2.10",
-    "dotf": "^1.2.0",
-    "execa": "^2.0.3",
-    "inquirer": "^6.4.1",
-    "insight": "^0.10.3",
-    "mz": "^2.7.0",
-    "recursive-readdir": "^2.2.2",
-    "rimraf": "^2.6.3",
-    "shelljs": "^0.8.3",
-    "terminal-link": "^1.3.0",
-    "uuid": "^3.3.2"
-  },
-  "devDependencies": {
-    "@types/argparse": "^1.0.36",
-    "@types/bluebird": "^3.5.27",
-    "@types/cli-spinner": "^0.2.0",
-    "@types/execa": "^0.9.0",
-    "@types/inquirer": "^6.0.3",
-    "@types/insight": "^0.8.0",
-    "@types/jest": "^24.0.15",
-    "@types/node": "^12.6.1",
-    "@types/recursive-readdir": "^2.2.0",
-    "@types/rimraf": "^2.0.2",
-    "@types/terminal-link": "^1.1.0",
-    "@types/uuid": "^3.4.5",
-    "jest": "^24.8.0",
-    "prettier": "^1.18.2",
-    "ts-jest": "^24.0.2",
-    "tslint": "^5.18.0",
-    "typescript": "^3.5.3"
-  }
+  "name": "data-studio-developer-tools",
```

Comment:
If we don't have any top level dependencies, we should just omit this file.

Reply:
doing that for now